packages/core/src/account/base.ts

```diff
@@ -99,7 +119,7 @@ export abstract class BaseSmartContractAccount<
     this.owner = params.owner;
 
     this.entryPoint = getContract({
-      address: params.entryPointAddress,
+      address: _entryPointAddress,
```

Comment:
```suggestion
      address: params.entryPointAddress,
```

Reply:
you mean `this.entryPointAddress` at @avasisht23 ?

Reply:
yup, corrected